test/unit/test_warnings.py

```diff
@@ -0,0 +1,20 @@
+import os
```

Comment:
Add copyright header

Reply:
Looks like none of the test files in `test/unit/` have copyright headers. Should they? (does test code need them in general?)

Reply:
In general yes, but I guess we can do that in bulk as as separate change.